README.md

```diff
@@ -13,7 +13,9 @@
 - **Important flags**<br>
 `-a` - display hidden files<br>
 `-R` - recursively displays files<br>
-`-l` - displays extra details like size, owner, group, date the file was last modified and permissions
+`-l` - displays extra details like size, owner, group, date the file was last modified and permissions<br/>
+
+`tree` - display files and directories as a tree structure.<br/>
```

Comment:
This should be `tree <path>` where the default path is current directory.

Reply:
corrected